src/VisualStudio/Core/Def/ColorSchemes/ColorSchemeApplier.cs

```diff
@@ -56,6 +61,11 @@ internal sealed partial class ColorSchemeApplier
         _settings = new ColorSchemeSettings(threadingContext, _serviceProvider, globalOptions);
         _colorSchemes = ColorSchemeSettings.GetColorSchemes();
         _classificationVerifier = new ClassificationVerifier(threadingContext, fontAndColorStorage, _colorSchemes);
+        _workQueue = new(
+            TimeSpan.FromMilliseconds(1000),
```

Comment:
try DelayTimeSpan.Idle

Reply:
💭 It seems like `Idle` could leave a really strange visual state, where `Short` would be equally effective but with less transition time.